files/generate_ansible_command.py

```diff
@@ -289,6 +302,9 @@ def get_hostname(x):
     else:
         commands_to_run.append('ansible-playbook -D %s' % p)
 
+for l in local_playbooks_to_run:
+    commands_to_run.append('sudo /usr/local/bin/ansible_local.py %s' % l)
```

Comment:
ho, so it seems it is to limit sudo, why not.
still having duplicate parameters is not nice but I guess it won't change very often.